config/base/api.yaml

```diff
@@ -56,6 +56,28 @@ spec:
             - name: tls
               mountPath: "/etc/tls"
               readOnly: true
+          startupProbe:
+            httpGet:
+              path: /healthz
+              port: 8080
+              scheme: HTTPS
```

Comment:
Does using HTTPS present an issue if the apiserver uses a certificate that is not globally trusted? This is the most likely deployment scenario, where the apiserver uses a cluster-signed/self-signed certificate, and the ingress re-encrypts the traffic.

Reply:
It doesn't create an issue, because the Kubelet sends an HTTPS request skipping the certificate verification.

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#http-probes